pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/globalcomputeforwardingrule/_http.log

```diff
@@ -676,17 +676,16 @@ X-Xss-Protection: 0
 
 ---
 
-POST https://compute.googleapis.com/compute/v1/projects/${projectId}/global/forwardingRules?alt=json
+POST https://compute.googleapis.com/compute/v1/projects/${projectId}/global/forwardingRules
 Content-Type: application/json
-User-Agent: Terraform/ (+https://www.terraform.io) Terraform-Plugin-SDK/2.10.1 terraform-provider-google-beta/kcc/controller-manager
+x-goog-request-params: project=${projectId}
 
 {
   "IPAddress": "0.0.0.0",
   "description": "A global forwarding rule",
   "labels": {
     "cnrm-test": "true",
-    "label-one": "value-one",
-    "managed-by-cnrm": "true"
```

Comment:
If we are going to support labels (which I guess we have to, because it's a beta resource, then I guess we have to do the managed-by-cnrm label)